.github/workflows/macos.yml

```diff
@@ -41,6 +41,7 @@ jobs:
         run:  |
           brew install llvm@16
           brew install conan@2
+          brew install suite-sparse
```

Comment:
This currently doesn't seem to work,
I will try to figure out how to do this on our Mac.
What we also could do is to compile GraphBLAS alongside QLever, but only if
it is explicitly requested.